src/Store/VimStoreConnector.re

```diff
@@ -13,6 +13,27 @@ module Model = Oni_Model;
 module Log = Core.Log;
 module Zed_utf8 = Core.ZedBundled;
 
+let findHighlightList = () => {
```

Comment:
did you run the formatter?

Reply:
can you hint me how to do that?

Reply:
You can use `esy format` and it will run the default formatter tool over your code.

That should then let it pass the bit the CI failed on.